dispatcher/dispatcher.py

```diff
@@ -76,69 +85,76 @@ def _accept_client_conn(self, server):
         # Register socket for receiving data
         self.selector.register(up_client_socket, selectors.EVENT_READ, self._receive_from_up_client)
 
-    def _receive_from_up_client(self, up_client_socket):
+    def _receive_from_up_client(self, up_client_socket: socket.socket):
         """
         Callback function for receiving data from up-client sockets.
 
         :param up_client_socket: The client socket.
         """
         try:
 
-            recv_data = up_client_socket.recv(BYTES_MSG_LENGTH)
+            recv_data: bytes = up_client_socket.recv(BYTES_MSG_LENGTH)
```

Comment:
Python is dynamically typed, meaning that you don't need to declare the type of a variable explicitly. But anyway, it depends on your preference. You can ignore it, I prefer adding types explicitly only in method parameters or return types.

Reply:
Got it. We're trying to be as explicit as possible, but fair point.